Appraisals

```diff
@@ -1,18 +1,19 @@
-appraise 'spree-3-1' do
-  gem 'spree', '~> 3.1.0'
+appraise 'spree-3-7' do
+  gem 'spree', '~> 3.7.0'
+  gem 'rails-controller-testing'
 end
 
-appraise 'spree-3-2' do
-  gem 'spree', '~> 3.2.0'
+appraise 'spree-4-0' do
+  gem 'spree_core', '~> 4.0.0'
```

Comment:
why you changing from `spree` to `spree_core` this is some particular reason to do that ?